integrationRest/pom.xml

```diff
@@ -17,7 +17,10 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.release>11</maven.compiler.release>
-    <maven-versions-rules>file:///${project.parent.basedir}/rules.xml</maven-versions-rules>   
+    <maven-versions-rules>file:///${project.parent.basedir}/rules.xml</maven-versions-rules>
+	<hamcrest.version>2.2</hamcrest.version>
```

Comment:
Best to add version to the main pom file and then use variables. Otherwise the versions needs to be changed on several places.

BTW latest Spring Boot is now 2.7.9

Reply:
Done 👍🏻